config.py

```diff
@@ -1,2 +1 @@
-
-API_TOKEN = 'token'
+API_TOKEN = '5961570363:AAHrpDN9V0Ti0vXKKuSLWn1ez0cdCxsOfUA'
```

Comment:
Your credentials have been compromised.

Reply:
You need to generate new credentials as this one is exposed to public.

Reply:
yeah